service.bal

```diff
@@ -146,7 +150,12 @@ service /admin on new http:Listener(9090) {
     # A resource for removing an AidPackage-Item
     # + return - aidPackageItem
     resource function delete aidpackages/[int packageID]/aidpackageitems/[int packageItemID]() returns int|error {
-        check deleteAidPackageItem(packageID, packageItemID);
+        AidPackageItem aidPackageItem = check getAidPackageItem(packageItemID);
+        if (check checkAlreadyPledgedAgainstAidPackageUpdate(aidPackageItem,true)) {
+            check deleteAidPackageItem(packageID, packageItemID);
+        } else {
+            return error 'error("Already done Pledges amount exceeds the Aid Package Item Quantitiy Update");
```

Comment:
Seems to be the case where other errors are constructed too, but this error that's being constructed is the builtin `error`, right? Any reason to not do

```suggestion
            return error("Already done Pledges amount exceeds the Aid Package Item Quantitiy Update");
```
?

Reply:
agreed to the suggestion. May have copied it from somewhere.